### DISCUSSION TITLE
Welcome to pythonproject Discussions!

### DISCUSSION THREAD

**Discussion by ML-Engineer12** [ML-Engineer12/pythonproject#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by israr96418** [comment]
I watched your python project
that you are done email
thats great

**Reply by ML-Engineer12** [reply]
thank you

**Comment by israr96418** [comment]
but i would suggest you to study different email library such "MIMEText MIMEMultipart" that help you create fancy email

**Reply by ML-Engineer12** [reply]
ok i will do it 
and share my progress
